apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -16,6 +15,13 @@ interface ILayoutContext {
     addNotification?: (notification: ITopNotification) => void
 }
 
+const detectMobileNavigator = (): boolean => {
```

Comment:
we have two places where this function is used, other one is at `apps/condo/domains/user/components/containers/AuthLayoutContext.tsx`

Reply:
Yes. I've fix one. Another is not a part of condo app